config/vufind/CookieConsent.yaml

```diff
@@ -0,0 +1,141 @@
+#
+#     Name              A regular expression to match cookie names that should be
+#                       cleared
+#
+# Iframe service configuration (see the VuFind wiki for an example:
+# https://vufind.org/wiki/configuration:cookie_consent)
+#
+#   EmbedUrl          Template for embedded content url. {data-id} can be used as a
+#                     placeholder for the data-id attribute of the tag.
+#   ThumbnailUrl      Template for thumbnail image url. {data-id} can be used as a
+#                     placeholder for the data-id attribute of the tag.
+#   IframeAllow       allow attribute for the iframe
+#   Description       Description text displayed alongside buttons to display the
+#                     content
+#   AllowOnceText     Text for the "Allow once" button
+#   AllowAlwaysText   Text for the "Allow always" button (remembers the choice)
+#
+# You can use e.g. https://cookiedatabase.org/ to find out the cookies used by most
+# popular services.
+#
+# All textual strings are translatable. The following placeholders can be used in
+# settings and translated texts:
+#
+# {{consent_cookie_name}}         Name of the consent cookie (default cc_cookie)
+# {{consent_cookie_expiration}}   Consent cookie expiration time
+# {{current_host_name}}           Server name of the VuFind server
+# {{vufind_cookie_domain}}        Domain for VuFind's cookies
+# {{vufind_session_cookie}}       VuFind session cookie name
+#
+CookieConsent:
+  #CookieExpiration: 365
+  Categories:
+    essential:
+      Title: CookieConsent::essential_cookies_title_html
+      Description: CookieConsent::essential_cookies_description_html
+      DefaultEnabled: true
+      Essential: true
+      Cookies:
+        - Name: '{{consent_cookie_name}}'
+          Domain: '{{vufind_cookie_domain}}'
+          Description: CookieConsent::cookie_description_consent_html
+          Expiration: '{{consent_cookie_expiration}}'
+          ExpirationUnit: days
+        - Name: 'cc_*'
+          Domain: '{{vufind_cookie_domain}}'
+          Description: CookieConsent::cookie_description_consent_html
+          Expiration: '{{consent_cookie_expiration}}'
+          ExpirationUnit: days
+        - Name: language
+          Domain: '{{vufind_cookie_domain}}'
+          Description: CookieConsent::cookie_description_language_html
+          Expiration: session
+        - Name: sidefacet-* (session storage)
+          Domain: '{{vufind_cookie_domain}}'
+          Description: CookieConsent::cookie_description_sidefacet_html
+          Expiration: session
+        - Name: '{{vufind_session_cookie}}'
+          Domain: '{{vufind_cookie_domain}}'
+          Description: CookieConsent::cookie_description_session_html
+          Expiration: session
+        - Name: ui
+          Domain: '{{vufind_cookie_domain}}'
+          Description: CookieConsent::cookie_description_ui_html
+          Expiration: session
+        - Name: vufind_cart, vufind_cart_src
```

Comment:
Is comma-separated list of names something else that needs to be documented?

Reply:
Yes, in the essence that it's free text.